src/Microsoft.DotNet.Build.Tasks/Targets/packageresolve.targets

```diff
@@ -34,23 +42,73 @@
       $(ResolveAssemblyReferencesDependsOn);
       ResolveNuGetPackages;
     </ResolveAssemblyReferencesDependsOn>
+    <NugetTargetFrameworkMoniker Condition="'$(NugetTargetFrameworkMoniker)' == ''">$(TargetFrameworkMoniker)</NugetTargetFrameworkMoniker>
   </PropertyGroup>
 
   <Target Name="ResolveNuGetPackages" 
           Condition="'$(ResolveNuGetPackages)'=='true'">
 
-    <ResolveNuGetPackages PackagesConfigs="$(ProjectPackagesConfigFile)"
+    <!-- Use old task for packages.config -->
+    <ResolveNuGetPackages Condition="'$(ProjectPackagesConfigFile)' != ''"
+                          PackagesConfigs="$(ProjectPackagesConfigFile)"
                           PackageRoot="$(PackagesDir)"
                           Platform="$(PlatformTarget)"
                           Configuration="$(Configuration)"
                           Language="$(Language)"
-                          TargetFramework="$(TargetFrameworkMoniker)"
+                          TargetFramework="$(NugetTargetFrameworkMoniker)"
                           TargetPlatformMoniker="$(TargetPlatformMoniker)">
 
       <Output TaskParameter="ResolvedAnalyzers" ItemName="Analyzer" />
       <Output TaskParameter="ResolvedReferences" ItemName="Reference" />
       <Output TaskParameter="ResolvedCopyLocal" ItemName="None" />
     </ResolveNuGetPackages>
+
+    <PropertyGroup>
+      <!-- Temporary workaround -->
```

Comment:
What is this working around? Can you make sure there is an issue tracking this if it is just temporary.